wit/world.wit

```diff
@@ -1,4 +1,4 @@
-package fermyon:spin@3.0.0;
+package spin:spin@3.2.0;
```

Comment:
```suggestion
package spin:up@3.2.0;
```

This may be a bit "cute" but I think it accurately reflects the use of this package, which is describing the world supported by `spin up`.

Reply:
Ooh I like `spin:up` (and agree with your objections to `spin:spin`).  Thank you!